src/geointerface.jl

```diff
@@ -1,3 +1,15 @@
+using Extents
+
+lookup_method = Dict(
```

Comment:
This could be `const`

Reply:
`const Dict{DataType, Function}`

Reply:
Resolved by https://github.com/yeesian/ArchGDAL.jl/pull/290/commits/3079fd31974a6ef5ccbf4241ee4b5bc2c4bfca2b